chia/server/node_discovery.py

```diff
@@ -707,7 +707,7 @@ def __init__(
         )
 
     async def start(self) -> None:
-        self.initial_wait = 60
+        self.initial_wait = 1
```

Comment:
@fchirica is this fine? I remember there was controversy around this number

Reply:
this is wallet only btw